### PR TITLE
feat(file imports): adds an experimental IFC parser

### PR DESCRIPTION
**Pull request by adamhathcock**
* experimental IFC parser can be toggled via feature flag, `featureFlags.fileImportIFCDotNetEnabled` in Helm Chart or `FF_FILEIMPORT_IFC_DOTNET_ENABLED` environment variable on the fileimport service in Docker Compose or local development environment.
* Experimental IFC parser is built in .Net
* File Import service Docker image is now based on Ubuntu and not Debian (distroless).
* Example IFC files are now located in `packages/fileimport-service/ifc-dotnet/ifcs`

### CODE REVIEW

File: packages/fileimport-service/Dockerfile [hunk 1]
```diff
@@ -19,79 +23,46 @@ ENV WAIT_VERSION=${WAIT_VERSION}
...
-
-# Prefixing PATH with our virtual environment should seek required binaries
-# from virtual environment first.
-# Unsetting python home
-ENV PATH=/venv/bin:${PATH} \
-    PYTHONHOME=
+COPY --from=dotnet-build-stage /app/output packages/ifc-converter
 
-ENTRYPOINT [ "tini", "--", "/nodejs/bin/node", "--no-experimental-fetch", "src/daemon.js"]
+ENTRYPOINT [ "./tini", "--", "node", "--no-experimental-fetch", "packages/fileimport-service/src/daemon.js"]
```
[the remark this answers is not in the document]
Comment (adamhathcock) [reply packages/fileimport-service/Dockerfile:76]: the dir tini was put in was never added to the path, not sure how it worked.

the working directory I set for tini means I had to change the path for the daemon

File: packages/fileimport-service/Dockerfile [hunk 2]
```diff
@@ -1,97 +1,74 @@
...
+
+RUN apt-get update -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     --no-install-recommends \
+     curl=8.5.0-2ubuntu10.5 \
+    && curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     --no-install-recommends \
+     nodejs=18.20.5-1nodesource1 \
+    && npm install -g yarn@1.22.22 \
```
Comment (gjedlicska) [inline packages/fileimport-service/Dockerfile:28]: this is not ok, we're not using yarn 1

we're using corepack enabled yarn in the workspace root

**Review by gjedlicska — CHANGES_REQUESTED** [review]

**Comment by adamhathcock** [comment]
@iainsproat I think I fixed everything about docker....need confirmation of the corepack change

**Comment by adamhathcock** [comment]
Two of the example files are failing on the step parser.  I think it's because of whitespace/separator intolerance.

**Comment by adamhathcock** [comment]
Need @gjedlicska to approve and merge

**Review by gjedlicska — APPROVED** [review]

**Pull request merged**